scripts/submitSparkPi.sh

```diff
@@ -45,4 +45,12 @@ docker run --rm --network host $IMAGE_NAME \
     --conf spark.armada.scheduling.nodeUniformity=armada-spark \
     --conf spark.armada.scheduling.nodeSelectors=armada-spark=true \
     --conf spark.armada.pod.labels=foo=bar \
+    --conf spark.armada.driver.limit.cores=200m \
+    --conf spark.armada.driver.limit.memory=450Mi \
+    --conf spark.armada.driver.request.cores=200m \
+    --conf spark.armada.driver.request.memory=450Mi \
+    --conf spark.armada.executor.limit.cores=100m \
+    --conf spark.armada.executor.limit.memory=510Mi \
+    --conf spark.armada.executor.request.cores=100m \
+    --conf spark.armada.executor.request.memory=510Mi \
```

Comment:
We are mixing Kubernetes bytes notation (`450Mi`) and Spark bytes notation (`450m` or `450mb`). See `DEFAULT_SPARK_EXECUTOR_MEMORY` and `DEFAULT_MEM`. Any thoughts?

Reply:
So I believe we are doing exactly what spark on k8s does.  Most of the notations are k8s podspec format.  The one different one is the SPARK_EXECUTOR_MEMORY param, which is actually a JVM parameter used in the k8s entrypoint.sh script like so: https://github.com/apache/spark/blob/master/resource-managers/kubernetes/docker/src/main/dockerfiles/spark/entrypoint.sh#L97-L98

For now, since it is what is used for spark on k8s, I'd like to leave it, but I will clarify that in the arch doc.  Is that ok for now?